frontend/openapi/generated/pacta/services/DefaultService.ts

```diff
@@ -71,13 +63,12 @@ export class DefaultService {
      * Deletes a pacta version by ID
      * deletes a single pacta version based on the ID supplied
      * @param id ID of pacta version to delete
-     * @returns EmptySuccess pacta version deleted successfully
-     * @returns Error unexpected error
+     * @returns any pacta version deleted successfully
      * @throws ApiError
      */
     public deletePactaVersion(
         id: string,
-    ): CancelablePromise<EmptySuccess | Error> {
+    ): CancelablePromise<any> {
```

Comment:
I get that this is the outcome of us not having an explicit return type for empties, I don't love this though. I'd far prefer we even return `cancelablePromise<void>`. Is that possible? I just really don't want `any`s floating around our codebase.